src/coreclr/debug/daccess/datatargetadapter.cpp

```diff
@@ -131,6 +131,14 @@ DataTargetAdapter::GetPlatform(
         _ASSERTE_MSG(false, "Not supported platform.");
         return E_NOTIMPL;
 
+#ifndef HOST_WINDOWS
+    // In windows, IMAGE_FILE_MACHINE_RISCV64 is not declared yet.
+    case IMAGE_FILE_MACHINE_RISCV64:
+        ulExpectedPointerSize = 8;
+        platform = CORDB_PLATFORM_POSIX_RISCV64;
+    break;
+#endif // HOST_WINDOWS
+
```

Comment:
```suggestion
    case IMAGE_FILE_MACHINE_RISCV64:
        ulExpectedPointerSize = 8;
        platform = CORDB_PLATFORM_POSIX_RISCV64;
    break;

```
And instead add the following somewhere (src/coreclr/inc/clrint.h may be a good place):
```
#ifndef IMAGE_FILE_MACHINE_RISCV64
#define IMAGE_FILE_MACHINE_RISCV64           0x5064  // RISCV64
#endif
```

Reply:
Thank you!